be/src/column/type_traits.h

```diff
@@ -300,6 +300,11 @@ struct ColumnTraits<int8_t> {
     using ColumnType = Int8Column;
 };
 
+template <>
```

Comment:
why add a uint8 specialization

Reply:
in order to match the following template function for uint8:
template <class T, typename = std::enable_if<is_type_complete_v<typename ColumnTraits<T>::ColumnType>, void>>
    Status do_visit(const FixedLengthColumn<T>& col)

Reply:
same question. it's only for unsigned int type, i'm afraid nobody needs i

Reply:
Got your point. just my test case cover this type. But it doesn't hurt anyway, right?